src/api/app/policies/event_subscription_policy.rb

```diff
@@ -0,0 +1,31 @@
+class EventSubscriptionPolicy < ApplicationPolicy
+  def initialize(user, record, opts = {})
+    super(user, record, opts.merge(ensure_logged_in: true))
+  end
+
+  def index?
+    record_of_user_and_group?(record)
```

Comment:
This code path is not used (yet) but the idea is to add meaningful policies that could be reused across the codebase, in order to achieve this I've added what (I think) is the expected logic for this Policy.

Please double check this because I've just inferred this from the codebase without any prior domain knowledge 🤷‍♂️